### PR TITLE
Manned turrets are now anchored

### PR DESCRIPTION
**Pull request by improvedname**
theres some things that should be able to happen instead of a random greyshit running into your base and running off with it

:cl: Improvedname
tweak: Manned guns are now anchored
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:)SOME RANDOM SHIT RAN INTO MY TURF AND STOLE MUH TURRET

### CODE REVIEW

File: code/game/machinery/porta_turret/portable_turret.dm [hunk 1]
```diff
@@ -942,6 +942,7 @@
 	icon_state = "machinegun"
 	can_buckle = TRUE
 	density = TRUE
+	anchored = 1
```
Comment (lzimann) [inline code/game/machinery/porta_turret/portable_turret.dm]: `TRUE`

**Comment by evsey9** [comment]
does this mean you can anchor/unanchor manned turrets now?

**Comment by Thunder12345** [comment]
No they're anchored forever

**Comment by Robustin** [comment]
At least add an unanchoring mechanism, i.e. wrenching

FYI they already anchor when manned, if you can't properly defend your base I'm not sure why a greyshit shouldn't be able to run off with it. But for other balance purposes I'm ok with this if you can add the wrench anchoring code used elsewhere in our code.

**Comment by jughu** [comment]
Ill get it on tommorow unlesd somebody wants to rush it

**Comment by Bawhoppen** [comment]
just make them have a slowdown when dragging

**Comment by Shadowlight213** [comment]
Are they anchored if someone is on it?
Could give someone hyperzine and have then drag a gunner for high speed fighting if not.

**Comment by Lumipharon** [comment]
or... just make them wrenchable with a delay?

**Comment by improvedname** [comment]
Im auctally having more fun this way

**Pull request closed without merging**